spec/opencost-specv01.md

```diff
@@ -207,7 +206,19 @@ Workloads are defined as entities to which Asset Costs are committed. Some resou
   </tr>
 </table>
 
-
+The following workload cost aggregations are supported in a complete implementation in the OpenCost Spec: 
+
+* container
+* pod
+* deployment
+* statefulset
+* job
+* controller
+* controller kind
+* label
```

Comment:
This would aggregate based on value after a users has provided a key, e.g. environment in the example below. 

```
"environment" : "dev", "environment" : "qa", "environment" : "production"
```

Would you see a use case for supporting #2 or #3?